src/client/components/DisplayMeals.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './home.css';
+import { FancyBorder } from './FancyBorder';
+
+export const DisplayMeals = ({ title }) => {
+    const singleTitle = title.map((item, index) => {
```

Comment:
I think `title` here could be a better name as if I read `title` I would expect to be a string not an array of data. Maybe consider a more fitting name like `meals`